Contact-form

tingamapuro04 opened the pull request:
I was working on the work branch where I created a last container => l-container where I had two sections text part on top and a form to take user input at the bottom with a button for submission. Inside the form I have two inputs, one is of type text and the other is of type email. I have not included any kind of validation for the two. I then have a text-area where some text can be added to.

Basir-Mohammadi committed:
Add Ellipse image to html file

Basir-Mohammadi committed:
Add ellipse image to the photo folder

Basir-Mohammadi committed:
Add ellipse style to the css file

tingamapuro04 committed:
Added general container and restructure the first container

tingamapuro04 committed:
Added image files to the photos directory

shaqayq committed:
change textbox style

shaqayq committed:
change submit button style

tingamapuro04 committed:
Added two image files

tingamapuro04 committed:
Added placeholders inside input tags

tingamapuro04 committed:
Added two background images to the l-container

tingamapuro04 committed:
Reduced the number of rows in the text-area

tingamapuro04 committed:
Linter changes

tingamapuro04 committed:
Update names of contributors

tingamapuro04 commented:
In this latest pull request I have worked with two other people to build the contact form

topeogunleye commented on index.html:
```diff
@@ -14,256 +14,258 @@
-                <p>Skills</p>
-                <img src="/photos/separator.png" alt="">
+            <div class="form">
+                <form action="GET">
```
- [ ] I kindly suggest that you kindly ensure your action method should equal "POST" and please register on form spree, then get your  unique id so that you can add it to your action value 

### EXAMPLE
`<form action="POST" action="https://formspree.io/f/{form_id}">
`

Please also ensure that your form submission works after adding your unique id from formspree and POST method.

topeogunleye commented on index.html:
```diff
@@ -14,256 +14,258 @@
-                <img src="/photos/separator.png" alt="">
+            <div class="form">
+                <form action="GET">
+                    <input type="text" name="name" placeholder="Merianne">
```
- [ ] KIndly ensure that your name has a `maxlength="30"` attribute. This will stop users from writing a name that's too long.

topeogunleye commented on index.html:
```diff
@@ -14,256 +14,258 @@
+                <form action="GET">
+                    <input type="text" name="name" placeholder="Merianne">
+                    <br>
+                    <input type="email" name="email" placeholder="natajayanj14@gmail.com">
```
- [ ] Kindly ensure that your email input has a required attribute. This will add form validation to ensure the user fills in an email

topeogunleye commented on index.html:
```diff
@@ -14,256 +14,258 @@
+                    <br>
+                    <input type="email" name="email" placeholder="natajayanj14@gmail.com">
+                    <br>
+                    <textarea name="name"  cols="38" rows="10">Write your message here</textarea>
```
- [ ] Kindly ensure that your textarea input has a required attribute. This will add form validation to ensure the user fills information  into the text area
- [ ] Kindly ensure that your textarea has a `maxlength="500"` attribute. This will stop users from filling in too much info here

topeogunleye commented on index.html:
```diff
@@ -14,256 +14,258 @@
+                    <br>
+                    <textarea name="name"  cols="38" rows="10">Write your message here</textarea>
+                    <div class="submit">
+                        <button type="submit">Get It Touch</button>
```
- [ ] I kindly suggest that you follow the template design for your button

| Your Button Design | Template Design Button |
| ------------- | ------------- |
| ![Screenshot from 2022-05-04 15-14-27](https://user-images.githubusercontent.com/52934128/166700343-86cfb98a-4e2c-4c72-9a8c-c1d8cba96fda.png) | ![Screenshot from 2022-05-04 15-13-06](https://user-images.githubusercontent.com/52934128/166700134-7963bbca-90fc-4ebc-995d-f366056234a1.png) |

- [ ] I kindly suggest that you add the different button interactions including hover, active (when it's pressed), disabled, and enable (default state) as shown in the Figma template below:

![Screenshot from 2022-04-27 19-12-50](https://user-images.githubusercontent.com/52934128/165592912-7d8bf224-8e6b-467d-bb19-cf63cd2b40fd.png)

the button on the pressed state has the `background: #2230D2;` and on hover has the `background: #6070FF;`

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #2230D2;
}
```
**for hover state**
```
button:hover {
    background-color: #6070FF;
}
```

topeogunleye reviewed (changes requested):
> # Changes Required 
> ### Hi @tingamapuro04,
> Nice job, you are almost there.
> 
> ![](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## (Highlights) Good Points: 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
>- [ ] Kindly ensure you follow the design template in Figma as much as possible, this includes the font, colours, images and spacing (margin/padding)

| Your Design  | Template Design |
| ------------- | ------------- |
| ![Screenshot from 2022-05-04 15-18-33](https://user-images.githubusercontent.com/52934128/166701270-f17f7745-be41-4c70-adac-c27956f1e85b.png)  | ![Screenshot from 2022-05-04 15-18-11](https://user-images.githubusercontent.com/52934128/166701216-c3e1e7e0-5e43-4868-a628-374e5b7e0a91.png)  |

> ## [Optional] suggestions:
> * Nothing to mention.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

tingamapuro04 committed:
Performed changes on the l-container and the form

tingamapuro04 committed:
Linter corrections

tingamapuro04 commented:
I have done changes on my work branch where the changes include;
1. I have changed my form method to "POST" and included a formspree link in my form action and confirmed that it is working perfectly.
2. I have added the attribute required and max-length of 30 to the input type="text".
3. I have added attribute required to the input type="email".
4. I have added attribute max-length=500 and attribute required to the text-area.
5. I have reduced the width of the button and enabled attributes such as hover and active

OmarRamoun reviewed (approved):
# 🟢 Project Approved! 🏆

## Hi @tingamapuro04 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ implemented previous code review requested changes
- ✔️ descriptive commit messages
- ✔️ descriptive PR title and description
- The code is top notch! 💯 and have a huge potential.

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. So, you can skip them if you don't have the time but remember to come back and implement them because they are going to enhance the quality of your code_

You can also consider:
- [ ] [OPTIONAL] Please, make sure that you stick to the figma design 🖼️ (in color, font, margins ...etc) as possible as listed in the project requirements.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
- This is My **GitHub**: [![@mrramoun](https://img.shields.io/github/followers/MrRamoun?label=Ramoun&style=social)](https://github.com/mrramoun)
- This is My **Twitter**: [![@ramoun16](https://img.shields.io/twitter/follow/ramoun16?label=ramoun16&style=social)](https://twitter.com/ramoun16)
- This is My **Slack**: [![Ramoun](https://img.shields.io/badge/slack-@Ramoun-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

topeogunleye commented:
> I have done changes on my work branch where the changes include;
> 
> 1. I have changed my form method to "POST" and included a formspree link in my form action and confirmed that it is working perfectly.
> 2. I have added the attribute required and max-length of 30 to the input type="text".
> 3. I have added attribute required to the input type="email".
> 4. I have added attribute max-length=500 and attribute required to the text-area.
> 5. I have reduced the width of the button and enabled attributes such as hover and active

Great Job. Keep it going! :smile:
